frontend/store/auctions.ts

```diff
@@ -239,6 +239,7 @@ export const actions = {
             alternativeDestinationAddress,
         }: { id: string; marketId: string; alternativeDestinationAddress: string | undefined }
     ) {
+        console.info(marketId); // to be removed after core implementation
```

Comment:
Please remove. Also, in the future please prepend comments to yourself with `TODO:` (then it can be easily found)